content/blog/hello-world.md

```diff
@@ -1,28 +1,45 @@
 +++
 title = "Hello world"
-date = 2022-01-19T14:28:05+01:00
+date = 2022-04-04T14:28:05+01:00
 description = "Introducing scverse"
 draft = false
 +++
 
-With the increasing reliance of scientific research on infrastructural and analytical software, foundational tools in life sciences have outgrown individual labs and institutes in terms of their scale and impact. That has encouraged us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences.
+With the increasing reliance of scientific research on infrastructural and analytical software, foundational tools in life sciences have outgrown individual labs and institutes in terms of their scale and impact. That has encouraged us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem.
 
-Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our scope and focus in the very beginning of our journey that we have called {{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}}.
+Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our initial scope and focus in the very beginning of our journey. We are excited to announce **scverse** – a consortium for joint maintenance of core packages in omics data analysis in life sciences.
 
+## What tools are part of scverse?
 
-## What this is about
+We start with the two most important data structures, [AnnData](https://github.com/scverse/anndata) and [MuData](https://github.com/scverse/mudata), together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are looking forward to expanding with more major libraries and frameworks which fit our vision.
```

Comment:
I would start with some context, also as to why we picked these tools. This paragraph is a bit abrupt IMO